src/main.rs

```diff
@@ -0,0 +1,65 @@
+mod config;
+mod client;
+
+use std::{env, process};
+
+fn main() {
+    let cfg = match config::read() {
+        Ok(cfg) => cfg,
+        Err(_) => {
+            println!("missing config, path: {}\n
+                     rerun `lmz configure`", config::path());
+
+            process::exit(1);
+        },
+    };
```

Comment:
As of [1.61](https://blog.rust-lang.org/2022/05/19/Rust-1.61.0.html#custom-exit-codes-from-main) `main` can return exit codes: `fn main() -> std::process::ExitCode`, might be something cool to play around with if you plan to have a variation of exit statuses.